ethcore/src/miner/miner.rs

```diff
@@ -614,12 +615,19 @@ impl Miner {
 							}
 						}).unwrap_or(default_origin);
 
+						// try to install zgp-checker before appending transactions
```

Comment:
it's highly unlikely that it will ever be changed once set. if it is, then it is not unreasonable to expect miners to restart in order to get the "correct" ZGP addresses. as such i don't see it as a huge problem.